src/main/javascript/src/pages/application/view/view.html

```diff
@@ -7,7 +7,57 @@ <h1>Applications</h1>
 				<p>Put your student application display here!</p>
 			</div>
 			<div if.bind="isProf">
-				<p>Put your prof application display here!</p>
+				<h4>Applications Requesting you as Supervisor</h4>
+				<p if.bind="requested.length === 0">No applications at this time</p>
+				<div  if.bind="requested.length > 0">
+					<div repeat.for="application of requested">
+						<p>
+							${application.applicant.firstName} ${application.applicant.lastName}: ${application.degree} in ${application.department}
+							<button click.delegate="toggleDetails(application.detailToggleId)">details</button>
+						</p>
+						<p>
+							<button click.delegate="setApplicationStatus(application.id, 'APPROVEDFUNDING')">approve with funding</button>
+							<button click.delegate="setApplicationStatus(application.id, 'APPROVEDNOFUNDING')">approve no funding</button>
+							<button click.delegate="setApplicationStatus(application.id, 'DENIED')">deny</button>
+						</p>
+						<div show.bind="application.showDetails">
+							<p>Term: ${application.term.season} ${application.term.year}</p>
+							<p>Status: ${application.status}</p>
+							<p>GPA: ${application.gpa}</p>
+							<p>Resume: ${application.resumeFileName}</p>
+							<p>Interests:</p>
+							<ul>
+								<li repeat.for="interest of application.applicant.interests">${interest.keyword}
+							</ul>
+						</div>
+					</div>
```

Comment:
Sections that show applications like this one would be perfect if it filtered the applications according to status.
When I click approve/deny, something should happen to tell me that my button press did something. Perhaps remove the application from the page entirely or move it to another queue that tells me my work here is done.
If you want to filter stuff in ES6+ style JS, we can say `let filteredApps = applications.filter(application => { return <condition> });` where <condition> can be like... `application.status === "STATUS" || application.status === "something else"`.

We will have to grab the applications again once we do this. An alternative, optimistic approach is setting the status in the front end, filtering, and silently update the database, only displaying an error if the request actually fails. Then, we would have to correct the blind, front end filtering by re-pulling the data from the db.

Reply:
there's one or two things in here that would help the others do application stuff, so I think I will merge this and do that filtering in a separate issue